api/alerts.go

```diff
@@ -108,14 +108,13 @@ func (a *Alerts) Update(viewName string, newAlert *Alert) (*Alert, error) {
 	return &alert, nil
 }
 
-func (a *Alerts) Add(viewName string, newAlert *Alert) (*Alert, error) {
```

Comment:
Other projects can depend on the api package from this project. This is a public API and should not be changed.

Reply:
Good point